farm.go

```diff
@@ -18,3 +25,11 @@ func main() {
 		log.Fatal("ListenAndServe: ", err)
 	}
 }
+
+func connectDB(c *app.Context) {
```

Comment:
This could possibly also be a method on the `app.Context`. I'd probably like it better this way because it changes the struct and it's better for functions to have no side effects. It's a total nit though.